whatwg.org/index

```diff
@@ -1,39 +1,93 @@
+             
+               <div class="span-all">
+                     <h2>The Standards</h2>
+                     <a class="standard" href="https://html.spec.whatwg.org/multipage/"><svg class="arrow" width="77" height="44" viewBox="0 0 77 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                          <title>Group</title>
+                          <g id="C" transform="translate(202 164)">
+                          <g id="G">
+                          <g id="V">
+                          <use xlink:href="#thingy" transform="matrix(-1 0 0 1 -125 -164)"/>
+                          </g>
+                          </g>
+                          </g>
+                          <defs>
+                          <path id="thingy" fill-rule="evenodd" d="M 16.4309 26.8506L 25.2302 35.7195C 27.1096 37.6138 27.1096 40.685 25.2302 42.5793C 23.3508 44.4736 20.3037 44.4736 18.4243 42.5793L 1.40955 25.4299C 0.46985 24.4827 -7.68049e-16 23.2414 -1.02865e-22 22C -5.00901e-16 20.7586 0.46985 19.5173 1.40955 18.5701L 18.4243 1.4207C 20.3037 -0.473568 23.3508 -0.473568 25.2302 1.4207C 27.1096 3.31498 27.1096 6.3862 25.2302 8.28047L 16.4309 17.1494L 72.1875 17.1494C 74.8454 17.1494 77 19.3211 77 22C 77 24.6789 74.8454 26.8506 72.1875 26.8506L 16.4309 26.8506Z"/>
+                          </defs>
+                          </svg>
+                          <h3>Read the HTML Living Standard</h3></a>
+                     <a class="standard" href="https://spec.whatwg.org/"><svg class="arrow" width="77" height="44" viewBox="0 0 77 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                          <title>Group</title>
+                          <g id="Can" transform="translate(202 164)">
+                          <g id="Gr">
+                          <g id="Vect">
+                          <use xlink:href="#path0_fill" transform="matrix(-1 0 0 1 -125 -164)"/>
+                          </g>
+                          </g>
+                          </g>
+                          <defs>
+                          <path id="path0_fill" fill-rule="evenodd" d="M 16.4309 26.8506L 25.2302 35.7195C 27.1096 37.6138 27.1096 40.685 25.2302 42.5793C 23.3508 44.4736 20.3037 44.4736 18.4243 42.5793L 1.40955 25.4299C 0.46985 24.4827 -7.68049e-16 23.2414 -1.02865e-22 22C -5.00901e-16 20.7586 0.46985 19.5173 1.40955 18.5701L 18.4243 1.4207C 20.3037 -0.473568 23.3508 -0.473568 25.2302 1.4207C 27.1096 3.31498 27.1096 6.3862 25.2302 8.28047L 16.4309 17.1494L 72.1875 17.1494C 74.8454 17.1494 77 19.3211 77 22C 77 24.6789 74.8454 26.8506 72.1875 26.8506L 16.4309 26.8506Z"/>
+                          </defs>
+                          </svg>
+                          <h3>See the other standards developed at the WHATWG</h3></a>
+               </div>
+               <hr>
+               <div class="grid">
+                     <a href="/faq"><h2>FAQ</h2></a>
```

Comment:
These aren't h2s; they are just links. (They are not headings for a section or article.)